pages/pil/modules/index.js

```diff
@@ -17,24 +16,6 @@ module.exports = settings => {
   //   '{{static.content.pages.profile.invite}}'
   // ]));
 
-  // app.use('/', (req, res, next) => {
-  //   // const establishment = req.user.profile.establishments.find(e => e.id === req.establishment);
-  //   // res.locals.static.establishment = establishment;
-  //   next();
-  // });
-
-  // app.post('/', (req, res, next) => {
-  //   const values = req.session.form[req.model.id].values;
-  //   const opts = {
-  //     method: 'POST',
-  //     headers: { 'Content-type': 'application/json' },
-  //     body: JSON.stringify(values)
-  //   };
-  //   return req.api(`/establishment/${req.establishment}/invite-user`, opts)
-  //     .then(() => next())
-  //     .catch(next);
-  // });
-
   app.post('/', (req, res, next) => {
     const values = req.session.form[req.model.id].values;
     console.log('TRAINING MODULES FORM VALUES ', JSON.stringify(values));
```

Comment:
Can we kill this while the code is at hand?

Reply:
done.